.rubocop_local.yml

```diff
@@ -0,0 +1,17 @@
+#
+# Overrides
+#
+GlobalVars:
+  AllowedVariables:
```

Comment:
Not sure you can leave this blank like this...you can just remove this entire overrides section entirely